Home_Work_11/task4.py

```diff
@@ -0,0 +1,68 @@
+# Создайте класс Матрица. Добавьте методы для: - вывода на печать,
+# сравнения,
+# сложения,
+# *умножения матриц
+
+from random import randint as rnd
+
+
+class Matrix:
+    list_matrix = []
+    min_value = 1
+    max_value = 20
+
+    def __init__(self, columns, rows):
+        self.columns = columns
+        self.rows = rows
+        self.list_matrix = [[rnd(self.min_value, self.max_value) for i in range(self.columns)] for i in range(self.rows)]
+
+    def __str__(self):
+        txt = ''
+        for i in range(len(self.list_matrix)):
+            for j in range(len(self.list_matrix[i])):
+                txt += f'{self.list_matrix[i][j]} '
+            txt += '\n'
+        return txt
```

Comment:
Можно попробовать так. Без конкатенации
```suggestion
        return '\n'.join(' '.join(map(str, line)) for line in self.list_matrix)
```